test/Core.Test/NotificationHub/NotificationHubPushNotificationServiceTests.cs

```diff
@@ -20,10 +20,10 @@ public class NotificationHubPushNotificationServiceTests
     [Theory]
     [BitAutoData]
     [NotificationCustomize]
-    public async void PushSyncNotificationAsync_Global_NotSent(
+    public async void PushNotificationAsync_Global_NotSent(
```

Comment:
```suggestion
    public async Task PushNotificationAsync_Global_NotSent(
```
I know we have this changed in the other PR. 
Not sure what ordering we will merge these in, but let's make sure `async Task` is the end state.

Reply:
Fixed in #5057
I have added order numbers to the PR's, should be more clear on the order now.